geometry_msgs/Vector3 best_view
---
bool is_finished